quipucords/api/tests_report.py

```diff
@@ -10,95 +10,94 @@
-                                              cpu_core_per_socket=1,
-                                              cpu_siblings=1,
-                                              cpu_hyperthreading=False,
-                                              cpu_socket_count=2,
-                                              cpu_core_count=2,
-                                              date_anaconda_log='2017-07-18',
-                                              date_yum_history='2017-07-18',
-                                              etc_release_name=os_name,
-                                              etc_release_version=version,
-                                              etc_release_release=release,
-                                              virt_virt='virt-guest',
-                                              virt_type='vmware',
-                                              virt_num_guests=1,
-                                              virt_num_running_guests=1,
-                                              virt_what_type='vt')
-            fact_collection.facts.add(fact)
-
-        fact_collection.save()
-        return fact_collection
-
-    def test_get_report_list(self):
-        """Create fact collection object via API."""
-        url = '/api/v1/reports/'
-
-        # Create a system fingerprint via fact collection receiver
-        self.create_fact_collection(
-            os_versions=['7.4', '7.4', '7.5'])
-
-        # Query API
-        response = self.client.get(url)
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
-        report_list = response.json()
-        self.assertIsInstance(report_list, list)
-        self.assertEqual(len(report_list), 1)
-        self.assertEqual(report_list[0]['report'][0]['count'], 2)
-        self.assertEqual(report_list[0]['report'][1]['count'], 1)
-
-    def test_get_fact_collection_report(self):
-        """Create fact collection object via API."""
-        url = '/api/v1/reports/'
-
-        # Create a system fingerprint via collection receiver
-        self.create_fact_collection(
-            os_versions=['7.4', '7.4', '7.5'])
-
-        # Query API
-        response = self.client.get(url, {'fact_collection_id': 1})
-        self.assertEqual(response.status_code, status.HTTP_200_OK)
-        report = response.json()
-        self.assertIsInstance(report, dict)
-        self.assertEqual(report['report'][0]['count'], 2)
-        self.assertEqual(report['report'][1]['count'], 1)
-
-    def test_get_fact_collection_404(self):
-        """Create fact collection object via API."""
-        url = '/api/v1/reports/'
-
-        # Create a system fingerprint via collection receiver
-        self.create_fact_collection(
-            os_versions=['7.4', '7.4', '7.5'])
-
-        # Query API
-        response = self.client.get(url, {'fact_collection_id': 2})
-        self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
+# import uuid
```

Comment:
Is there an issue to re-implement these tests?

Reply:
I plan to clean these up in #348